### ISSUE TITLE
Configure NFS ... Operation not permitted

### ISSUE THREAD

**Issue by v-vovk** [sergeycherepanov/homebrew-docker-virtualbox#1]
my log:
```
[1;34m[INFO] [0mConfigure NFS ... 
[0;33m
 !!! Sudo will be necessary for editing /etc/exports !!! [0m
cat: /etc/exports: No such file or directory
tee: /etc/exports: Operation not permitted

```

it happened when I make this command:
```
brew services start docker-virtualbox 
```

**Comment by lucaspdude** [comment]
How did you manage to solve this?

**Comment by chanmathewrbc** [comment]
@wolfykey how did you solve this? I'm running into same issue all of a sudden...

Update: For some reason after updating my Motherboard BIOS SVM was disabled, I had to re-enable it.
I also needed to run the following commands again to update permissions:

First:
`sudo touch /etc/exports`

Second:
```
sudo tee /etc/sudoers.d/docker-machine-nfs  <<SUDOERS
%staff ALL=(ALL) NOPASSWD: /sbin/nfsd
%staff ALL=(ALL) NOPASSWD: /bin/cp /etc/nfs.conf /etc/nfs.conf.bak
%staff ALL=(ALL) NOPASSWD: /usr/bin/tee /etc/exports
%staff ALL=(ALL) NOPASSWD: /usr/bin/tee /etc/nfs.conf
%staff ALL=(ALL) NOPASSWD: $(brew --prefix docker-virtualbox)/bin/gobetween
SUDOERS
```
Third:
`docker-machine-init initialize` again to fix permissions

At which point I ran into a certificate error from logging at my logs, and to fix that I needed to run:
`docker-machine regenerate-certs docker` - `docker` being the name of the container

Then everything worked. Hope this helps someone!
